t/jv-oneof.t

```diff
@@ -23,29 +23,50 @@ $schema = {oneOf => [{type => 'object'}, {type => 'string', multipleOf => 3}]};
 validate_ok 13, $schema, E('/', '/oneOf Expected object/string - got number.');
 
 $schema = {oneOf => [{type => 'object'}, {type => 'number', multipleOf => 3}]};
-validate_ok 13, $schema, E('/', '/oneOf/1 Not multiple of 3.');
+validate_ok 13, $schema, E('/', '/oneOf/0 Expected object - got number.'),
+  E('/', '/oneOf/1 Not multiple of 3.');
 
 # Alternative oneOf
-# http://json-schema.org/latest/json-schema-validation.html#anchor79
+# https://json-schema.org/draft-07/json-schema-validation.html#rfc.section.7
```

Comment:
👍

Reply:
It took a bit of sleuthing to figure out what part of the spec was being referenced there. I only figured it out when I broken the 'format' tests and found the bit in the spec explaining how it worked differently than everything else -- I had altered `schema_type` to infer `string` from `format`, and then found the tests that broke as a consequence.